src/js/player.js

```diff
@@ -2763,8 +2773,14 @@ class Player extends Component {
    * and calls `reset` on the tech`.
    */
   reset() {
+    if (this.tech_) {
+      this.tech_.clearTracks(['text']);
+    }
     this.loadTech_(this.options_.techOrder[0], null);
     this.techCall_('reset');
+    if (isEvented(this)) {
+      this.trigger('vjs_reset');
```

Comment:
I think the best name for this may be `playerreset`. We already have precedent with the [`playerresize` event](https://github.com/videojs/video.js/pull/4864).

Reply:
ok